docs/content/bundle/create.md

```diff
@@ -0,0 +1,199 @@
+
+🙏🏼 Please, [do not embed bash commands] directly in the porter.yaml file because it is much more difficult to get the escaping and quotes correct than putting the bash in a separate file.
+
+Bundles actions should be [idempotent], meaning an action can be repeated without causing errors, and it should have the same effect each time.
+A user of your bundle should be able to re-run install/upgrade/uninstall multiple times in a row, perhaps because the bundle failed half-way through when run the first time.
+Make sure that your commands gracefully handle resources already existing in the install and upgrade actions, and handles already deleted resources in the uninstall action.
+Some mixins, like the exec mixin, have [built-in support for error handling][ignore-errors] and ignoring errors in certain circumstances.
+
+## Test Your Bundle
+
+After you have finished editing the porter.yaml, repeat the `porter build` command to re-build the bundle with your latest changes.
+Then run `porter install mybundle --force` to install the bundle.
+The \--force flag is only safe to use in development, and it allows you to incrementally develop a bundle and re-install it without having to first uninstall it and start over after every change.
+
+## Publish Your Bundle
+
+When you are ready to share your bundle with others, the next step is to publish it to a registry.
+Most registries work with Porter, if you run into trouble check our list of [compatible registries]. 
+
+The [porter publish] command by default pushes the bundle to the registry defined in the porter.yaml file.
+
+1. Edit the registry field and change it to a registry that you can push to.
+   For example, if you have an account on Docker Hub, change the registry value from localhost:5000 to your Docker Hub username.
+2. Edit the name field and change it to your preferred name for the bundle, like "mybundle".
+3. Use the docker login command to first authenticate to the destination registry:
+   ```
+   docker login REGISTRY
+   ```
+
+   For example, if the registry defined in the porter.yaml is ghcr.io/myuser, then run `docker login ghcr.io` to authenticate. 
+
+   If you are publishing to Docker Hub, the registry field in the bundle would just be your Docker Hub username, and you would authenticate to the registry with just `docker login` without any additional arguments.
+   This works because the Docker client by default uses Docker Hub (docker.io) when a registry is not fully specified.
+4. Now, publish the bundle by running `porter publish`.
+
+## Use the Published Bundle
+
+Once your bundle is published, people can use it by setting the `--reference` flag on relevant porter commands to the bundle's reference.
+The name, registry, and version fields are used to generate the bundle's default publish location when porter publish is run.
+Unless you specify additional flags during publish, the bundle is published to REGISTRY/BUNDLE_NAME:vBUNDLE_VERSION.
```

Comment:
Maybe it would be helpful to link to those additional flags from here?

Reply:
I'll link to the publish command help text 👍